lib/rules/no-duplicate-selectors/__tests__/index.js

```diff
@@ -84,6 +84,12 @@ testRule(rule, {
     message: messages.rejected("a b"),
     line: 1,
     column: 12,
+  }, {
+    code: "a { & {} }",
+    description: "duplicate caused by &-parent selector",
+    message: messages.rejected("a"),
```

Comment:
Wait ... why is this a duplicate? Don't we end up with `a` and `a a`, which are different selectors?